websites/A/ArtStation/presence.ts

```diff
@@ -0,0 +1,80 @@
+const presence = new Presence({
+    clientId: "917417055458852865"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "artstation",
+    startTimestamp: browsingTimestamp
+  };
+
+  if (document.location.pathname.startsWith("/messages"))
+    presenceData.details = "Checking inbox";
+  // (not functional)
+    // else if (document.querySelector("user-profile")) {
+    // presenceData.details = "Viewing a profile";
+    // presenceData.state = document.querySelector('meta[property="og:title"]').getAttribute("content");
+  // }

```

Comment:
Why?

Reply:
The only differentiation between a normal page and an artist's profile that I could find is this Class name part and my `querySelector` can't actually detect it and thus labels it as a normal page
![image](https://user-images.githubusercontent.com/77577746/144961601-bda27d52-159a-43ff-8731-b0eb50e07ba0.png)
I left it as a comment in the code hoping somebody else can find a fix for it, or I can delete it if it's not allowed/unnecessary

Reply:
Done